### DISCUSSION TITLE
[Config Support]: Clarification on detector config width & height

### DISCUSSION THREAD

**Discussion by dougransom** [blakeblackshear/frigate#13913] [Config Support] [answered]
### Describe the problem you are having

I read in one place that if the width and height are not specified, they default to to default width and height and use ffmmpeg to transcode, rather than using the dimensions of the input stream.  In other places it says it says it uses the dimensions of the input stream for detection by default.

```    detect:
      enabled: true # <---- disable detection until you have a working camera feed
      width: 640 # <---- update for your camera's resolution
      height: 480 # <---- update for your camera's resolution
```

### Version

0.14.1

### Frigate config file

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

**Comment by NickM-27** [comment, marked as answer]
in the current version of frigate, it will use the native resolution of the stream that is provided
